.github/workflows/rubocop.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: Rubocop
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+      TESTOPTS: '-v'
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Ruby 3.2
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.2
+          bundler-cache: true
+      - name: bundle install
+        run: bundle install --jobs 4 --retry 3
```

Comment:
Any reason to run `bundle install` explicitly? Per my understanding `setup-ruby` does that when `bundler-cache` is enabled.

Reply:
This was a copy-paste. Can remove it in a separate PR.

Reply:
Removed it lets see what happens...

Reply:
looks good now :muscle: